consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/CommitPayload.java

```diff
@@ -26,8 +26,8 @@
 import java.util.Objects;
 import java.util.StringJoiner;
 
-public class CommitPayload implements Payload {
-  private static final int TYPE = QbftV1.COMMIT;
+public class CommitPayload extends QbftPayload {
+  public static final int TYPE = QbftV1.COMMIT;
```

Comment:
why does this need to be public? couldn't see where it is used

Reply:
yup - should be private, was part of a secondary piece of work I was tinkering with